modules/os2forms_digital_post/src/Form/SettingsForm.php

```diff
@@ -258,8 +258,27 @@ public function submitForm(array &$form, FormStateInterface $formState): void {
   private function testCertificate(): void {
     try {
       $certificateLocator = $this->certificateLocatorHelper->getCertificateLocator();
-      $certificateLocator->getCertificates();
-      $this->messenger()->addStatus($this->t('Certificate succesfully tested'));
+      $certificatePath = $this->settings->getCertificate()[CertificateLocatorHelper::LOCATOR_TYPE_FILE_SYSTEM]['path'];
```

Comment:
if the returned array is empty, you will get a warning here - because value is not set. Might be wise to handle that gracefully